File: cleanup_logo.py
import os
import re

dir_path = r"C:\Users\elpap\Downloads\nuevo programa de dolarito\project\src"
logo_regex = re.compile(r'<img[^>]*Diseno_sin_titulo\.png[^>]*>', re.IGNORECASE | re.MULTILINE | re.DOTALL)
new_logo = r'<span className="font-extrabold text-xl sm:text-2xl tracking-tight bg-clip-text text-transparent bg-gradient-to-r from-blue-400 to-indigo-500">SISTEMA DE EVENTOS</span>'

for root, dirs, files in os.walk(dir_path):
    for f in files:
        if f.endswith('.tsx'):
            path = os.path.join(root, f)
            with open(path, 'r', encoding='utf-8') as file:
                content = file.read()
            
            if 'Diseno_sin_titulo' in content:
                new_content = logo_regex.sub(new_logo, content)
                with open(path, 'w', encoding='utf-8') as file:
                    file.write(new_content)
                print(f"Reemplazado en {f}")

File: cleanup_logo.mjs
import fs from 'fs';
import path from 'path';
import { fileURLToPath } from 'url';

const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);

const dirPath = path.join(__dirname, 'src');
const logoRegex = /<img[^>]*Diseno_sin_titulo\.png[^>]*>/gi;
const newLogo = '<span className="font-extrabold text-xl tracking-tight bg-clip-text text-transparent bg-gradient-to-r from-blue-400 to-indigo-500">SISTEMA DE EVENTOS</span>';

function walkDir(dir) {
  fs.readdirSync(dir).forEach(file => {
    let fullPath = path.join(dir, file);
    if (fs.statSync(fullPath).isDirectory()) {
      walkDir(fullPath);
    } else if (fullPath.endsWith('.tsx')) {
      let content = fs.readFileSync(fullPath, 'utf8');
      if (content.includes('Diseno_sin_titulo')) {
        let newContent = content.replace(logoRegex, newLogo);
        fs.writeFileSync(fullPath, newContent, 'utf8');
        console.log(`Replaced in ${file}`);
      }
    }
  });
}

walkDir(dirPath);
